docs/platform/connector-development/connector-builder-ui/incremental-sync.md

```diff
@@ -92,18 +111,43 @@ The `from-date` is set to the cutoff date of articles synced already and the `to
 In some cases, it's helpful to reference the start and end date of the interval that's currently synced, for example if it needs to be injected into the URL path of the current stream. In these cases it can be referenced using the `{{ stream_interval.start_time }}` and `{{ stream_interval.end_time }}` [placeholders](/platform/connector-development/config-based/understanding-the-yaml-file/reference#variables). Check out [the tutorial](./tutorial.mdx#adding-incremental-reads) for such a case.
 :::
 
-## Incremental sync without time filtering
+## API time filtering capabilities
```

Comment:
This section should be removed as this went away with the switch to SchemaForm.

Now, data feed is a toggle in the Advanced section called `Whether the target API is formatted as a data feed`